backend/driver.pm

```diff
@@ -98,7 +98,7 @@ sub start {
 
             $self->{backend}->run(fileno($process->channel_in), fileno($process->channel_out));
             _exit(0);
-    }, sleeptime_during_kill => .1)->blocking_stop(1)->separate_err(0)->subreaper(1)->start;
+    }, sleeptime_during_kill => 10)->kill_sleeptime(0)->blocking_stop(1)->separate_err(0)->subreaper(1)->start;
```

Comment:
I think you found out by now that this is reverting a change I introduced to speedup the shutdown of processes. Your effect is also shown by tests when they time out.

Reply:
Yes, I've mentioned it also in the chat. The problem is that the stop function of Mojo::IOLoop::ReadWriteProcess simply makes no sense. You're either hammering the process with SIGTERM or wasting time.